packages/nexrender-database-redis/src/index.js

```diff
@@ -41,7 +41,11 @@ const insert = async entry => {
     entry.updatedAt = now;
     entry.createdAt = now;
     
-    await client.setAsync(`nexjob:${entry.uid}`, JSON.stringify(entry));
+    if (process.env.REDIS_TTL) {
+        await client.setAsync(`nexjob:${entry.uid}`, JSON.stringify(entry), 'EX', process.env.REDIS_TTL);
```

Comment:
Maybe worth adding a comment explaining that 'EX' is a redis flag meaning expire? Only recommending since I didn't know what it meant initially